CRM/Cplabels/Form/Search/Cplabels/Volunteer.php

```diff
@@ -58,18 +59,15 @@ public function buildForm(&$form) {
      */
     $form->assign('elements', array('team', 'serve_type', 'limit', 'correspondence_type'));
 
-    $form->addRule('team',E::ts('Field "Team" is required'),'required');
-    $form->addRule('serve_type',E::ts('Field "Service type" is required'),'required');
+    $form->addRule('team', E::ts('Field "Team" is required'), 'required');
+    $form->addRule('serve_type', E::ts('Field "Service type" is required'), 'required');
   }
 
   /**
    * Get a list of summary data points
    *
-   * @return mixed; NULL or array with keys:
-   *  - summary: string
-   *  - total: numeric
    */
```

Comment:
I'd like to keep the @return documentation here. Please see my comments above about the similar topic. Thanks!

Reply:
Fixed